examples/active-class-name/pages/[slug].js

```diff
@@ -0,0 +1,14 @@
+import { useRouter } from 'next/router'
+import Nav from '../components/Nav'
+
+const SlugPage = () => {
+  const { asPath } = useRouter()
+  return (
+    <>
+      <Nav />
+      <p>Hello, I'm the {asPath} page</p>
+    </>
+  )
+}
+
+export default SlugPage
```

Comment:
I think this code will cause a client mismatch (`asPath` is only available after the first render), so better to just say it's a dynamic page.

```suggestion
import Nav from '../components/Nav'

const SlugPage = () => (
  <>
    <Nav />
    <p>This is a dynamic route!</p>
  </>
)

export default SlugPage
```

Reply:
well I just used displayed the slug route... as that is what is important... that you can see that the slug is active.. i.e. more than one